src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1460,7 +1461,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return s_variablesMappings.GetValue(
                     Me,
                     Function(BoundUsing)
-                        Return New Variables(BoundUsing.ResourceList.As(Of IVariableDeclaration))
+                        Dim usingStatementSyntax = Syntax.ChildNodes().OfType(Of UsingStatementSyntax).SingleOrDefault()
+                        Return New Variables(BoundUsing.ResourceList.As(Of IVariableDeclaration), usingStatementSyntax)
```

Comment:
It seems to be a fairly easy change to make it conform to the interface docs, so I'm going with that route.